src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,96 @@
+package duke;
+
+import duke.tasks.TaskList;
+
+import java.io.*;
+
+/**
+ * Represents the file containing the list of tasks.
+ */
+public class Storage {
+    private static final String text = "./data/duke.txt";
+    private static BufferedWriter writer;
+
+    /**
+     * Takes in a task and adds it to the tasklist, while writing it to the file
+     * @param task Task ot be added.
+     * @throws IOException if an error while reading has occurred.
+     */
+    static void writeToFile(String task) throws IOException {
+        try {
+            Parser.parseTask(task);
+            if (task.startsWith("deadline") ||
+                    (task.startsWith("event") && task.contains("/at ")) ||
+                    (task.startsWith("todo"))) {
```

Comment:
Perhaps you can break down your Boolean checks into variables to avoid having a complicated expression!

Reply:
That makes sense, thank you!